packages/block-library/src/social-link/save.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+export default function save( { attributes } ) {
+	const { site, url } = attributes;
+
+	if ( ! url ) {
+		return null;
+	}
+
+	// TODO: suggestions for fall back icon?
+	// @todo: These classes don't appear to render in the final markup at all.
```

Comment:
Marcus, can you take a look? I think we might be able to remove both comments and the classname thing below.